tooling/index.rst

```diff
@@ -0,0 +1,10 @@
+==================
+ Advanced Tooling
+==================
```

Comment:
```suggestion
==============
Advanced Tools
==============
```

Shorter, simpler and more appropriate, since tooling implies a collection or system of tools/infrastructure, whereas these are individual, specific, niche tools.